frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -240,7 +241,9 @@ export const createResources = async (
   const requests: Promise<K8sResourceKind>[] = [];
   requests.push(createImageStream(formData, dryRun));
   if (!formData.serverless.enabled) {
-    requests.push(createDeploymentConfig(formData, dryRun));
+    if (!dryRun) {
```

Comment:
In this case `DeploymentConfig` would not be created for any of the import flows since by default we send `dryRun` as `true` for all the import flows.

Reply:
@rohitkrai03 I think first we pass true to perform dryRun and then we don't pass any and default value if false as in 

https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/import/ImportForm.tsx#L139

Am I missing something here?